### ISSUE TITLE
check_ceph_df :=> ValueError: could not convert string to float: TiB

### ISSUE THREAD

**Issue by amritdeep** [ceph/ceph-nagios-plugins#45]
check_ceph_df isn't able to convert Tib. I am getting following error :- 

/usr/local/libexec/nagios/sl_plugins/check_ceph_df --name client.nagios --keyring /etc/ceph/ceph.nagios.keyring -c /etc/ceph/ceph.conf -W 75 -C 90 -p rbd
Traceback (most recent call last):
  File "/usr/local/libexec/nagios/sl_plugins/check_ceph_df", line 179, in <module>
    sys.exit(main())
  File "/usr/local/libexec/nagios/sl_plugins/check_ceph_df", line 123, in main
    pool_usage_percent = float(poolvals[3])
ValueError: could not convert string to float: TiB

**Comment by valerytschopp** [comment]
What is your Ceph version?

**Comment by valerytschopp** [comment]
And on the same host, what is the output of:
```
/usr/bin/ceph --name client.nagios --keyring /etc/ceph/ceph.nagios.keyring -c /etc/ceph/ceph.conf df
```

**Comment by valerytschopp** [comment]
It related to #42

**Comment by ki4rbc** [comment]
This patch corrected the problem for me
```bash
--- /home/ubuntu/.local/src/ceph-nagios-plugins/src/check_ceph_df       2019-01-09 14:22:33.931590949 +0000
+++ /usr/lib/nagios/plugins/check_ceph_df       2019-01-09 14:38:16.759427782 +0000
@@ -137,7 +137,7 @@
             # print 'DEBUG:', globalvals
             # finally 4th element contains percentual value
             # print 'DEBUG USAGE:', globalvals[3]
-            global_usage_percent = float(globalvals[3])
+            global_usage_percent = float(globalvals[4])
             global_available_space = globalvals[1]
             global_total_space = globalvals[0]
             # print 'DEBUG WARNLEVEL:', args.warn
```

**Comment by valerytschopp** [comment]
Fixed with commit 9ac9de0510a8246b69c0d64a4e910ee9a5129636
